### ISSUE TITLE
Typo or bug?

### ISSUE THREAD

**Issue by mrwrock** [lucbettaieb/origami_bot#1]
https://github.com/lucbettaieb/origami_bot/blob/a4f3f4bc95e13a6a18113e272d25d2b245a03523/src/origami_main.cpp#L110

Looks like it should be l_enc_tick instead

**Comment by lucbettaieb** [comment]
Yep, you found a bug.  A lot of that class never got totally working, so it's very under-tested and old, stale code.

Can I ask what you're looking at it for?  Just wondering! :smile:

**Comment by mrwrock** [comment]
I was trying to figure out how to use wiringpi in Ros without root access to drive a RC servo. Turns out if you use wiringpisetupsys and a system() call you can control gpio without root access, but it takes too long to achieve a 50hz loop rate and you cannot control a servo so I'm going to have to just roslaunch as root and use loop_rate.sleep() (since the pwm function didn't run at the right frequency and I didn't have an oscilloscope to measure it's true frequency).

Your work looks great, I'm going to try and make a 2 wheeled balancing robot when my motors arrive so if you're interested I can share my work

Best regards,
Michael Wrock

On Oct 23, 2017 4:46 PM, Luc Bettaieb <notifications@github.com> wrote:

Yep, you found a bug. A lot of that class never got totally working, so it's very under-tested and old, stale code.

Can I ask what you're looking at it for? Just wondering! 😄

**Comment by lucbettaieb** [comment]
Ah cool!  Yeah I ended up just running everything from root.  You can do `sudo -s` to get into the root shell and then set up your ROS environment from there.  I'd love to see a video and the code if you get it working!  Good luck!
